.env

```diff
@@ -9,3 +9,5 @@ APPLICATION_THEME=orange
 
 #environment can be changed if it is toggled
 CREDENTIAL_REGISTRY_EDIT=true
+
+MAXIMUM_STORAGE_LIMIT_IN_MB=20
```

Comment:
adding MAXIMUM_STORAGE_LIMIT_IN_MB configuration in Inji.properties and using in Inji, makes application & configuration separate.

Reply:
Modified - fetching maximumStorageLimitInMB from configurations